### DISCUSSION TITLE
NATS compability

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [sidekiq/sidekiq#6233] [Ideas]
Hi,

may I ask what you think about NATS support? Redis clusters work pretty bad on kubernetes (tried various helm charts and was playing around, it was pretty easy to brick the clusters just by deleting pods etc - on reddit there are many people too which express the same experiences).

So it would be cool to have NATS support? I guess it's not so easy to implement, but would be still worthwile maybe?

**Comment by mperham** [comment]
Answered in the FAQ.

**Reply by avvari-da** [reply]
Can you drop this link here? Couldn't find in the FAQ.

**Reply by raybaxter** [reply]
https://github.com/sidekiq/sidekiq/wiki/FAQ#wouldnt-it-be-awesome-if-sidekiq-supported-mongodb-postgresql-mysql-sqs--for-persistence
